support-frontend/assets/helpers/abTests/__tests__/abtestTest.ts

```diff
@@ -337,69 +337,193 @@ describe('init', () => {
+					],
+					excludeContributionsOnlyCountries: true,
+				}),
+			};
+
+			const participations: Participations = abInit({
+				...abtestInitalizerData,
+				abTests,
+				selectedAmountsVariant: buildSelectedAmountsVariant(
+					contributionsOnlyAmountsTestName,
+				),
+			});
 
-		it(`does not assign a user to a test if excludeCountriesSubjectToContributionsOnlyAmounts is true set and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
+			expect(participations.t1).toBeUndefined();
+		});
+
+		it(`does assign a user to a test if excludeContributionsOnlyCountries is false and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
 			const abTests = {
 				t1: buildTest({
 					variants: [
 						buildVariant({ id: 'control' }),
 						buildVariant({ id: 'variant' }),
 					],
-					excludeCountriesSubjectToContributionsOnlyAmounts: true,
+					excludeContributionsOnlyCountries: false,
 				}),
 			};
 
 			const participations: Participations = abInit({
 				...abtestInitalizerData,
 				abTests,
-				selectedAmountsVariant,
+				selectedAmountsVariant: buildSelectedAmountsVariant(
+					contributionsOnlyAmountsTestName,
+				),
 			});
 
-			expect(participations).toEqual({});
+			expect(participations.t1).toBeDefined();
+		});
+
+		it(`does assign a user to the test if excludeContributionsOnlyCountries is true BUT selectedAmountsVariant test name is NOT ${contributionsOnlyAmountsTestName}`, () => {
+			const abTests = {
+				t1: buildTest({
+					variants: [
+						buildVariant({ id: 'control' }),
+						buildVariant({ id: 'variant' }),
+					],
+					excludeContributionsOnlyCountries: true,
+				}),
+			};
+
+			const participations: Participations = abInit({
+				...abtestInitalizerData,
+				abTests,
+				selectedAmountsVariant: buildSelectedAmountsVariant('foo'),
+			});
+
+			expect(participations.t1).toBeDefined();
+		});
+	});
+
+	describe('includeOnlyCountriesSubjectToContributionsOnlyAmounts', () => {
+		it(`does assign a user to a test if audience is CONTRIBUTIONS_ONLY and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
+			const abTests = {
+				t1: buildTest({
+					variants: [
+						buildVariant({ id: 'control' }),
+						buildVariant({ id: 'variant' }),
+					],
+					audiences: {
+						CONTRIBUTIONS_ONLY: {
+							offset: 0,
+							size: 1,
+						},
+					},
+					excludeContributionsOnlyCountries: false,
+				}),
+			};
+
+			const participations: Participations = abInit({
+				...abtestInitalizerData,
+				abTests,
+				selectedAmountsVariant: buildSelectedAmountsVariant(
+					contributionsOnlyAmountsTestName,
+				),
+			});
+			expect(participations.t1).toBeDefined();
+		});
+
+		it(`does assign a user to a test if excludeContributionsOnlyCountries is false, includeOnlyContributionsOnlyCountries is false and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}`, () => {
```

Comment:
This test description is referring to `includeOnlyContributionsOnlyCountries` which we're no longer implementing perhaps `does assign a user to a test if excludeContributionsOnlyCountries is false, and selectedAmountsVariant test name is ${contributionsOnlyAmountsTestName}` is more accurate?

Reply:
I'm not sure we need this test though, as we have the same test here: https://github.com/guardian/support-frontend/pull/6695/files#diff-ea6fffa4fcd929f02fe9c5d08ac836f2e162cd156cf20495c3656938686dc841R363

Reply:
I don't think we need this test tbh, as we have the same test here in the `excludeContributionsOnlyCountries` amounts describe block: I don't think we need this test tbh, as we have the same test here in the `excludeContributionsOnlyCountries` amounts describe block: https://github.com/guardian/support-frontend/pull/6695/files#diff-ea6fffa4fcd929f02fe9c5d08ac836f2e162cd156cf20495c3656938686dc841R363

Reply:
Agree removed this due to duplicate